pkg/menu/menu.go

```diff
@@ -1,15 +1,28 @@
 package menu
 
 import (
+	"fmt"
 	"io"
+	"strconv"
+	"strings"
 
 	ui "github.com/gizak/termui/v3"
 	"github.com/gizak/termui/v3/widgets"
 )
 
 const menuWidth = 50
 
-// newParagraph will return a paragraph object with given initial text.
+// Entry contains all the information needed for a boot entry.
+type Entry interface {
+	// Label is the string will show in menu.
+	Label() string
+	// IsDefault returns true means this entry is default.
+	// If there is many default Entrys, choose the the first in the list.
+	IsDefault() bool
+	Exec() error
```

Comment:
you'll need to add a comment for exec. 
golint or go vet should have told you, if they did not, sorry!